api/src/migrations/1645259040554-ImpactStoredFunctions.ts

```diff
@@ -258,8 +258,53 @@ export class ImpactStoredFunctions1645259040554 implements MigrationInterface {
     END;
   $$
   LANGUAGE plpgsql;
-
   `);
+
+    await queryRunner.query(`
+    CREATE OR REPLACE FUNCTION get_h3_material_table_column(h3DataId uuid)
```

Comment:
Do we really need this stored function? Looking at the logic I think it would be better to have this on the codebase, but at the same time, since this is for retrieving a map, but getting just a unique table and column name by a ID should be fairly fast